### PR TITLE
DISCO-72 update build

### PR DESCRIPTION
**Pull request by isGabe**
ticket: https://alleyinteractive.atlassian.net/browse/DISCO-72

There are a bunch of security vulnerabilities, and it made more sense to just update the build, rather than trying to audit and fix individual vulnerabilities on an ancient build.

### CODE REVIEW

File: client/config/webpack.config.js [hunk 1]
```diff
@@ -1,100 +1,58 @@
...
+            loader: 'postcss-loader',
+            options: {
+              postcssOptions: {
+                config: path.resolve(__dirname, '../config/postcss.config.js')
+              }
+            },
+          },
+          'css-loader',
+          'sass-loader',
+        ],
```
Comment (pattiereaves) [inline client/config/webpack.config.js]: Maybe this order? 
```suggestion
        use: [
          {
            loader: MiniCssExtractPlugin.loader,
            options: {
              publicPath: '../../',
            },
          },
          'css-loader',
          {
            loader: 'postcss-loader',
            options: {
              postcssOptions: {
                config: path.resolve(__dirname, '../config/postcss.config.js')
              }
            },
          },
          'sass-loader',
        ],
```

File: client/js/admin.js [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
  * @description JS manipulations to the fm-overlays admin area.
  *
  */
 
+ /**
+  * Set a condition field's wrapper label to the same value as the current option
+  * @param {node} selectEl a <select> element
+  */
+const setLabel = (selectEl) => {
+  const wrapper = selectEl.closest('.fm-fm_overlays_conditionals');
```
Comment (pattiereaves) [inline client/js/admin.js:17]: is selectEl a jquery function? (if so it's not obvious)

File: package.json [hunk 3]
```diff
@@ -4,9 +4,8 @@
   "description": "A Fieldmanager extension to render dynamic content in modal boxes/overlays on a site.",
   "private": true,
   "scripts": {
-    "watch": "npm run start",
-    "start": "webpack --config client/config/webpack.config.js -d --watch --colors",
-    "build": "webpack --config client/config/webpack.config.js -p"
+    "dev": "webpack --config client/config/webpack.config.js -d --mode development",
```
Comment (pattiereaves) [inline package.json]: it doesn't look like this does anything special based on mode from the webpack config, do you need it?

**Review by pattiereaves — APPROVED** [review]
🦄

File: client/js/admin.js (hunk 2 above)
Comment (isGabe) [reply client/js/admin.js:17]: No, that's just a param for the function, which is a select element. But maybe the naming could be improved.

File: package.json (hunk 3 above)
Comment (isGabe) [reply package.json]: No, you're right. I just copied this in at the start but didn't actually set up the dev task.

**Pull request merged**